utils/build-icons/index.js

```diff
@@ -1,6 +1,9 @@
+const Promise = require('bluebird');
+
 const generateTokenFile = require('./lib/generateTokenFile');
+const generateThemeableTokenFile = require('./lib/generateThemeableTokenFile');
```

Comment:
Making these two imports separately so in the future we can easily take one out if we no longer needs it.